src/intl/en/learn-quizzes.json

```
@@ -326,6 +326,51 @@
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
   "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "i001-prompt": "What is true about DAOs?",
+  "i001-a-label": "DAOs are collectively owned via governance tokens",
+  "i001-a-explanation": "DAOs are collectively owned, but that’s not the only correct statement.",
+  "i001-b-label": "They are governed by their members",
+  "i001-b-explanation": "DAOs are governed by their members, but that’s not the only correct statement.",
+  "i001-c-label": "They are working towards a shared mission",
+  "i001-c-explanation": "DAOs are working towards a shared mission, but that’s not the only correct statement.",
+  "i001-d-label": "All of the above",
+  "i001-d-explanation": "Correct, a DAO is a collectively-owned, blockchain-governed organization working towards a shared mission.",
+  "i002-prompt": "What are practical examples of how to use a DAO?",
+  "i002-a-label": "Decentralized protocols, members vote on the matters of the protocol or how to develop the product",
+  "i002-a-explanation": "Protocol DAOs are one example, but DAOs are not limited to that.",
+  "i002-b-label": "Collective ownership, e.g., for NFTs or physical assets",
+  "i002-b-explanation": "Collector DAOs are one example, but DAOs are not limited to that.",
+  "i002-c-label": "Ventures and grants, pool capital and vote on projects to fund",
+  "i002-c-explanation": "Venture or grants DAOs are one example, but DAOs are not limited to that.",
+  "i002-d-label": "All of the above",
+  "i002-d-explanation": "A DAO can have a multitude of 'missions'. Check out the resources to find practical examples.",
+  "i003-prompt": "Unlike traditional organizations, DAOs are…",
+  "i003-a-label": "Usually hierarchical",
+  "i003-a-explanation": "DAOs are usually flat, and fully democratized.",
+  "i003-b-label": "Transparent and fully public about their activities",
+  "i003-b-explanation": "Thanks to on-chain voting, decisions are transparent on the blockchain. Discussions and other elements of the decision-making process are usually open as well.",
+  "i003-c-label": "Controlled by a central party",
+  "i003-c-explanation": "Changes require voting by the members. Services offered are handled automatically in a decentralized manner.",
+  "i003-d-label": "Restricted regarding who can suggest changes",
+  "i003-d-explanation": "Usually, every DAO member can suggest changes.",
+  "i004-prompt": "What is essential about Smart Contracts for DAOs?",
```

Lowercase smart contract
```suggestion
  "i004-prompt": "What is essential about smart contracts for DAOs?",
```